SPECS/kernel/config

```diff
@@ -198,7 +206,7 @@ CONFIG_CGROUP_DEVICE=y
 CONFIG_CGROUP_CPUACCT=y
 CONFIG_CGROUP_PERF=y
 CONFIG_CGROUP_BPF=y
-# CONFIG_CGROUP_MISC is not set
+CONFIG_CGROUP_MISC=y
```

Comment:
Provides a controller for misc resources on the host
https://cateee.net/lkddb/web-lkddb/CGROUP_MISC.html